src/char/Baicang.py

```
@@ -0,0 +1,294 @@
+            return
+        deadline = self._now() + duration
+        while self._now() < deadline:
+            if not self.is_current_char or self.is_dead:
+                return
+            self.normal_attack()
+            remaining = deadline - self._now()
+            if remaining > 0:
+                self.sleep(min(self.NORMAL_ATTACK_INTERVAL, remaining))
+
+    def _right_click_burst(self, duration):
+        """持续右键点击 ``duration`` 秒，不管理方向键。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)